config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  root 'users#index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
```

Comment:
- [ ] It would be better if we fix the resource routing as suggested by the previous code reviewer. It will also help us make use of the nested routing mechanism

Reply:
i did make the changes

Reply:
Please take a look at the project again